pyproject.toml

```diff
@@ -128,6 +129,8 @@ extend-exclude = [
 'src/trio/lowlevel.py' = ['F401']
 'src/trio/socket.py' = ['F401']
 'src/trio/testing/__init__.py' = ['F401']
+'src/trio/_core/_generated_io_kqueue.py' = ['A002']
+'src/trio/_core/_io_kqueue.py' = ['A002']
```

Comment:
probably good to have comments here (for the F401 too) saying what the rule is and why it's ignored.

Reply:
Changed in 67116bc